assets/stylesheets/purple/main.scss

```diff
@@ -20,7 +20,9 @@ body {
   padding-top: 60px;
   .content {
     margin-bottom: 90px;
-    > a {
+    > p,
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4